fickling/context.py

```diff
@@ -1,23 +1,26 @@
 import pickle
 
-from fickling.hook import hook_pickle_load
 import fickling.loader as loader
+from fickling.analysis import Severity
+from fickling.hook import hook_pickle_load
 
 
 class FicklingContextManager:
-    def __init__(self, run_after_analysis=True, block=[3, 4, 5]):
-        # TODO Determine what the best defaults for block are
+    def __init__(self, max_acceptable_severity=Severity.LIKELY_UNSAFE):
```

Comment:
```suggestion
    def __init__(self, max_acceptable_severity=Severity.LIKELY_SAFE):
```
If the file is unsafe we do want to block it, don't we?